apps/platform/src/lib/clipboard.ts

```diff
@@ -0,0 +1,35 @@
+import { toast } from 'sonner'
+
+export function copyToClipboard(
+  message: string,
+  successMsg = 'Text copied to clipboard!',
+  errorMsg = 'Failed to copy text.'
+): void {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- navigator.clipboard is checked
+  if (navigator.clipboard) {
+    navigator.clipboard
+      .writeText(message)
+      .then(() => toast.success(successMsg))
+      .catch((error) => {
+        // eslint-disable-next-line no-console -- console.error is used for debugging
+        console.error(errorMsg, error)
```

Comment:
I would suggest to use `toast.error(errorMsg)` and then log the error in the console separately